ci/scripts/e2e-sink-test.sh

```diff
@@ -31,7 +31,8 @@ mv target/debug/risedev-dev-"$profile" target/debug/risedev-dev
 mv target/debug/librisingwave_java_binding.so-"$profile" target/debug/librisingwave_java_binding.so
 
 export RW_JAVA_BINDING_LIB_PATH=${PWD}/target/debug
-export RW_CONNECTOR_RPC_SINK_PAYLOAD_FORMAT=stream_chunk
+# TODO: Switch to stream_chunk encoding once it's completed, and then remove json encoding as well as this env var.
+export RW_CONNECTOR_RPC_SINK_PAYLOAD_FORMAT=json
```

Comment:
Note that here I switched back to JSON encoding because it looks not so easy to implement decoding for decimal & timestamp right now.